pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -121,6 +121,23 @@ type ExecCommand struct {
 	// Optional list of environment variables that have to be set
 	// before running the command
 	Env []EnvVar `json:"env,omitempty"`
+
+	// +optional
+	// Whatever the command needs should be executed on background.
+	// If "false" the controller should wait for command to finish and report its status.
+	// If "false" the controller won't wait for it to finish (command can run indefinitely or until it is manually stopped).
+	// Default value is "true" for group.kind: run, debug.
+	// Default value is "false" for group.kind: build, test.
+	// Default value is "false" for all commands without group.kind
+	Background bool `json:"background,omitempty"`
+
+	// +optional
+	// Whatever the command needs be restarted when files changed.
```

Comment:
```suggestion
	// Whatever the command needs should be restarted when files changed.
```
To match the phrasing above, though I feel like we could word this more clearly.